### ISSUE TITLE
What does a good guide have?

### ISSUE THREAD

**Issue by pocc** [kennethreitz/awesome-guides#3]
Some guides are AWESOME! And some ... aren't. 

### Qualities

I've come up with some things that I think make python-guide.org and postgresguide.org stand out:

- Comprehensive on $subject without being cumbersome
- Scoped clearly
- Searchable
- Skimmable
- Easy to contribute to
- Updated regularly
- TOC that is either a sidebar or easily accessible

Thoughts?

**Comment by kennethreitz** [comment]
Yes, I think a separate section for ones that aren't updated regularly may be in order.
